### PR TITLE
Splunk Integration

### PR DESCRIPTION
**Pull request by rcastley**
Hi,

Please see my PR for Splunk Observability Cloud integration (hopefully everything has been done correctly) :-)

Also, is there a way to define additional OTEL_RESOURCE_ATTRIBUTES? As I need to pass in `deployment.environment`.

Many thanks,
- Robert

### CODE REVIEW

**Review by rcastley — COMMENTED** [review]
Spelling fixes

File: openllmetry/integrations/splunk.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: "LLM Observability with Splunk and OpenLLMetry"
+sidebarTitle: "Datadog"
```
Comment (nirga) [inline openllmetry/integrations/splunk.mdx:3]:
```suggestion
sidebarTitle: "Splunk"
```

File: openllmetry/integrations/splunk.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Thirdly, make sure `otlp` is defined in the traces pipeline:
+
+```yaml
+  pipelines:
+    traces:
+      receivers: [jaeger, otlp, sapm, zipkin]
+      processors:
+      - memory_limiter
+      - batch
+      #- resource/add_environment
```
Comment (nirga) [inline openllmetry/integrations/splunk.mdx:44]: Is it needed?

**Review by nirga — COMMENTED** [review]
Thanks! I'll add support for setting arbitrary resource attributes! 
https://github.com/traceloop/openllmetry/issues/156

File: openllmetry/integrations/splunk.mdx (hunk 1 above)
Comment (rcastley) [reply openllmetry/integrations/splunk.mdx:3]: Thank you :-)

File: openllmetry/integrations/splunk.mdx (hunk 2 above)
Comment (rcastley) [reply openllmetry/integrations/splunk.mdx:44]: It's a direct copy/paste from Splunk configuration. Just want to ensure that `otlp` is in the receivers and that the exporter is using `sapm` to get the traces into Splunk Observability Cloud.

Comment (rcastley) [reply openllmetry/integrations/splunk.mdx:44]: Fire away 👍🏻

**Review by nirga — APPROVED** [review]
Very cool, thank you so much! 🙏

**Pull request merged**